Documentation/contributing/making-changes.rst

```diff
@@ -202,6 +202,56 @@ squash before submitting the Pull Request:
 
        $ git checkout my-branch
 
+#. Fetch the upstream code
+
+    .. code-block:: bash
+
+       $ git fetch upstream
+
+#. Rebase onto the upstream code
+
+   * Your favorite text editor will start and shows the commands for the interactive rebasing.
+   * Change the "pick" to "squash" for all the commits you want to squash.
+   * For the series of commits to squash, keep the first one with "pick".
+   * Once you save and exist your editor will start again to merge all the commit messages.
+   * Add a commit message that best describes your change then save and exit.
```

Comment:
I think you are confusing interactive with non-interactive rebase. The instruction below is correct but this is non interactive rebase. It simply moves your feature branch to be based on top of master. The text description speaks about interactive rebase which allows you to edit the history, for example for squashing. This should be actually part of the last point where you mention squashing.

Reply:
@v01d I'll delete this text.